### PR TITLE
[FIX] Send to the wrong address

### PR DESCRIPTION
**Pull request by tommasini**
All the information on https://github.com/MetaMask/mobile-planning/issues/472#issuecomment-1308689200

### CODE REVIEW

File: app/components/Views/SendFlow/Amount/index.js [hunk 1]
```diff
@@ -662,6 +662,10 @@ class Amount extends PureComponent {
       transactionObject.readableValue = value;
     }
 
+    if (selectedAsset.isETH) {
```
Comment (sethkfman) [inline app/components/Views/SendFlow/Amount/index.js:665]:
```suggestion
    if (selectedAsset.isETH) transactionObject.to = transactionTo;
```

Comment (sethkfman) [reply app/components/Views/SendFlow/Amount/index.js:665]: Not a blocker but I think this would accomplish the same thing with a couple less lines.

**Review by sethkfman — APPROVED** [review]
LGTM

File: app/components/Views/SendFlow/Amount/index.js (hunk 1 above)
Comment (tommasini) [reply app/components/Views/SendFlow/Amount/index.js]: Pushed suggestion

**Pull request merged**